dist/ng-wig.js

```diff
@@ -127,7 +131,9 @@ angular.module('ngWig').component('ngWig', {
           return;
         }
       }
+      _this.beforeExecCommand({ command: command, options: options });
```

Comment:
I needed this for the same use case described in the previous commit. The placeholders are contenteditable="false", because the user should not be able to change the displayed column name. However, we still want to be able to make them bold or italic. For this, they should be made contenteditable right before the command is executed, and then set back to false right after. I thought using hooks would be the best way to do this.

Again, this might be overcomplicating the ngWig editor and it might not be what you want.